src/request-data/resources.js

```diff
@@ -67,6 +67,10 @@ export default ({ i18n }, createResource) => {
 
   const formDraft = createResource('formDraft', () =>
     setupOption(data => shallowReactive(transformForm(data))));
+
+  // Form attachments (published)
+  createResource('formAttachments');
```

Comment:
I think this should be a local resource rather than one available app-wide. (I really need to work on #674 so that I can document the difference!) The other resources in this file are used in the router and other places that mean that they can't be created in components. But the default is for a resource to be created in a component (locally); it will be removed when the component is unmounted. The `useRequestData()` composable returns a `createResource` function that can be used to create a local resource.

There's a question though of which component should create the local resource. The most common choice would be `FormShow`, in which case `formAttachments` should be added to `useForm()` along with the other form subresources there. For example, that's where `formDatasetDiff` is created. The advantage of creating a resource in `FormShow` is that it will persist across tabs (across child routes). The downside is that because it persists, there's more opportunity for it to become out-of-sync with other data. For example, `formDatasetDiff` is reset after a form draft is published, and we'd probably need to do the same for `formAttachments`.

Alternatively, you could create the resource in `FormOverview` or even `FormOverviewRightNow`. It would be removed once the component is unmounted as the user navigates to a different page. That would entail less complexity (no keeping data in sync, also fewer things to test). But it would result in an additional request if the user navigates to the form overview, then another form page, then back to the form overview.

Curious what you think makes the most sense here!